waardepapieren-service/configuration/waardepapieren-config.json

```diff
@@ -17,7 +17,9 @@
   "PRODUCT_DESCRIPTION" : "Uittreksel Basis Registratie Persoonsgegevens",
   "PRODUCT_PURPOSE" : "Bewijs verblijfadres in woonplaats",
   "SOURCE_DATA_SELECTION" : [
-    {"Burgerservicenummer (BSN)" : "burgerservicenummer"},
-    {"Woonplaats verblijfadres" : "verblijfadres.woonplaats"}
+    {"Naam" : "_embedded.ingeschrevenpersonen[0].naam"},
```

Comment:
All except the BSN lead to objects, which makes this kind of ugly for demo. Can we keep this to non-object references until we've fixed the display on the PDF?

Reply:
Changed config accordingly. Config itself still looks ugly, but I guess that's ok for now. Pdf shown is fine now with this config.